### PR TITLE
Composing feedback

### PR DESCRIPTION
**Pull request by andreafioraldi**
Now FeedbacksTuple is not a thing anymore.
State take just one feedback for the normal feedback and one feedback as objective.
You can use multiple feedbacks composing them in a boolean formula.

For instance `feedback_and!(MyFeedback::new(), feedback_not!(CrashFeedback::new()))` returns an instance of AndFeedback now.

### CODE REVIEW

**Comment by andreafioraldi** [comment]
Bump to 0.2 because this and #84 break the old API

File: libafl/src/feedbacks/mod.rs [hunk 1]
```diff
@@ -47,77 +47,239 @@ where
...
-        self.1.discard_metadata_all(input)
+    ( $head:expr, $($tail:expr), +) => {
+        // recursive call
+        $crate::feedbacks::AndFeedback::new($head , feedback_and!($($tail),+))
+    };
+}
+
+/// Variadic macro to create a chain of OrFeedback
+#[macro_export]
+macro_rules! feedback_or {
```
Comment (domenukk) [inline libafl/src/feedbacks/mod.rs:244]: Maybe `feedbacks_or`?

(or even `feedbacks_any`?

Comment (andreafioraldi) [reply libafl/src/feedbacks/mod.rs:244]: I want to maintain the names consisent, and feedbacks_not does not make sense as it takes just one parameter

Comment (andreafioraldi) [reply libafl/src/feedbacks/mod.rs:244]: And in the end feedback_or creates a new OrFeedback, the name makes sense

**Pull request merged**